src/scorer.py

```diff
@@ -35,6 +37,7 @@ def score(self, cand_o: Optional[str]) -> Tuple[int, str]:
         if not cand_o:
             return Scorer.PENALTY_INF, ""
 
+        _, target_seq = self._objdump(self.target_o)
```

Comment:
Move this back into the constructor -- we want to avoid objdumping the target repeatedly